EmailNotifications/Email.cs

```diff
@@ -43,32 +44,49 @@ private static void SendCompletedCallback(object sender, AsyncCompletedEventArgs
 
         public void sendEmail()
         {
-            string[] styleProperties = { "color:red;padding-top: 40px", "color:red;font-weight:bold", "color:black;padding-top: 11px",
-                                            "color:red;padding-top: 35px","color:black","color:black;padding-top: 20px",
+            string[] styleProperties = { "color:red;padding-top: 40px;font-weight: bold", "color:red;font-weight:bold", "color:black;padding-top: 11px",
+                                            "color:red;padding-top: 35px;font-weight: bold","color:black","color:black;padding-top: 20px",
                                                "color:grey" ,"color:grey","color:red;padding-top: 24px","color:red;","color:grey;padding-top: 9px"};
             string[] fontSizeProperties = {"6" ,"3","3","6","4","6","4","4","2","1","3" };
-
+            string imgSrc = "logo.PNG";
+            //string path = Environment.GetFolderPath(Environment.SpecialFolder.MyPictures);
```

Comment:
Ghost Code

Reply:
@Hunter27 have you read the topics we created for you guys?

Reply:
once in a while why?

Reply:
No consistency on spacing as well.

Reply:
Ash appreciate the masterpiece